app/routes/meetings/publish/uittreksels/index.js

```diff
@@ -21,6 +21,7 @@ export default class MeetingsPublishUittrekselsRoute extends Route {
     const query = {
       include: 'behandeling.versioned-behandeling',
       filter: { zitting: { ':id:': this.meetingId } },
+      sort: params.sort,
```

Comment:
we're also not passing page and size, best to add those as well. 
+ we might want to check if this is also an issue on other routes.